File: testdata/InlineMethod/Java/current/Order.java
import java.util.List;

public class Order {
    private List<Item> items;

    public double calculateTotal() {
        double total = 0;
        for (Item item : items) {
            total += item.getPrice();
        }
        return total * 1.1;  // Apply tax
    }
}

File: testdata/InlineMethod/Cpp/current/Order.cpp
#include <vector>

class Item {
public:
    double getPrice() const { return price; }
private:
    double price;
};

class Order {
private:
    std::vector<Item> items;

public:
    double calculateTotal() {
        double total = 0;
        for (const auto& item : items) {
            total += item.getPrice();
        }
        return total * 1.1;  // Apply tax
    }
};
